src/main/java/com/uid2/admin/vertx/service/PartnerConfigService.java

```diff
@@ -71,6 +71,25 @@ public void setupRoutes(Router router) {
         }), Role.ADMINISTRATOR));
     }
 
+    @Override
+    public Collection<OperationModel> qldbSetup(){
+        try {
+            String config = partnerConfigProvider.getConfig();
```

Comment:
I think you could write one method that can handle all these cases, so the logic is all in one place.